README.md

```diff
@@ -354,11 +354,11 @@ easier.
 
   * `--no-pstate-check`: Do not crash out if Intel P-states are not disabled.
 
-  * `--hardware-reboots`: Restart physical hardware before each benchmark
-    execution.
-
 Note that you should not collect results intended for publication with
-development switches turned on.
+development switches turned on. We also recommend that for 'real'
```

Comment:
I think this is a litle hidden here. Shall we move it into `Step 6` with some notes on trying first without reboots? We also seem to not mention rc.local anywhere.

Sorry, I just realised that.

Reply:
OK, sounds like a good idea...